docs/guide/dev/features/mentions/README.md

```diff
@@ -14,4 +14,123 @@ For licensing, see LICENSE.md.
+
+You can create multiple mention instances attached to an editor with different markers.
+
+## Setting up synchronous data feed
+
+The easiest way to configure data feed is to provide an array of text matches. Mentions plugin will use synchronous data feed and create item IDs by itself.
+
+```javascript
+config.mentions = [ { feed: ['Anna', 'Thomas', 'John'] } ];
+```
+
+By default query matching for an array feed is case insensitive. You can change this behavior by setting {@linkapi CKEDITOR.plugins.mentions.configDefinition.caseSensitive caseSensitive} configuration option.
+
+## Setting up asynchronous data feed
+
+### Backend URL
+
+You can provide a backend URL string which will be used to fetch text matches from custom endpoint service. Each time when a user types matching text into an editor your backend service will be queried for text matches. Ajax URL request should response with an array of matches in JSON format.
+
+A backend URL string features `encodedQuery` special variable replaced with a mentions query. `encodedQuery` variable allows you to create customized URL e.g. for query `@anna` and the given URL `/users?name={encodedQuery}` your endpoint service will be queried with `/users?name=anna`.
+
+```javascript
+config.mentions = [ { feed: '/users?query={encodedQuery}' } ];
+```
+
+To avoid multiple HTTP requests to your endpoint service each HTTP response is cached by default and shared globally. To disable caching use {@linkapi CKEDITOR.plugins.mentions.configDefinition.cache cache} configuration option.
+
+### Asynchronous callback
+
+This method is recommended for advanced users who would like to take full control of the data feed. It allows you to provide data feed as an function which accepts two parameters: `options` and `callback`. Provided function will be called every time when user types matching text into an editor. The `options` object contains information about current query and a marker.
+
+Depending on your use case, you can use this code as an example boilerplate to create your own function feed:
+
+```javascript
+config.mentions = [ {
+	feed: function( opts, callback ) {
+		var xhr = new XMLHttpRequest();
+
+		xhr.onreadystatechange = function() {
+			if ( xhr.readyState == 4 ) {
+				if ( xhr.status == 200 ) {
+					callback( JSON.parse( this.responseText ) );
+				} else {
+					callback( [] );
+				}
+			}
+		}
+
+		xhr.open( 'GET', '/users?name=' + opts.query );
+		xhr.send();
+	}
+} ];
+```
+
+### Data items structure
+
+When using asynchronous method i.e. backend URL string or a callback function, you should provide correct object structure. See {@link guide/dev/features/autocomplete/README#data-callback autocomplete data callback guide} for more information about required item properties.
+
+## Customizing text matching
+
+You are not forced to stick to default mentions text matching. It could be customized by special marker character and a number of minimal characters required to start data querying.
```

Comment:
> It could be customized by special marker character and a number of minimal characters required to start data querying.
>
> You can easily configure this options by setting minChars and marker options.

->

You can customize the marker and minimal character count to your needs:

(both marker and min char count linked) as you did originally)

☝️ This will simplify it a little.